sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -43,7 +43,7 @@ export { AzureKeyCredential }
 
 // @public
 export type BeginCopyModelOptions = FormRecognizerOperationOptions & {
-    intervalInMs?: number;
+    updateIntervalInMs?: number;
```

Comment:
I can't find the PR that u have for the changelog, so just make sure this is there too

Reply:
Yes https://github.com/Azure/azure-sdk-for-js/pull/9117/commits/10558cc62d2fb94e040212b8600082f22b6d68f7